external/rootpy

```diff
@@ -1 +1 @@
-Subproject commit f0f123d5ebce3983ef3bd3d5d8c962787ca499b4
+Subproject commit 295bfb2624a3bf286348f61de009d0c0f3dfb323
```

Comment:
f0f123d5ebce3983ef3bd3d5d8c962787ca499b4 points to the rootpy version with my fix in it, but there are now newer versions.

Do this:

cd external/rootpy
git fetch origin
git pull upstream master
cd ../../
git diff external/rootpy


The change should then be to:

Subproject commit f946e870fb3b06c3bddbbd849ac13f111d771aa3

On 22/04/2016 09:22, DouglasBurns wrote:
>
> In external/rootpy 
> <https://github.com/BristolTopGroup/DailyPythonScripts/pull/318#discussion_r60705456>:
>
> > @@ -1 +1 @@
> > -Subproject commit f0f123d5ebce3983ef3bd3d5d8c962787ca499b4
> > +Subproject commit 295bfb2624a3bf286348f61de009d0c0f3dfb323
>
> i just checked out external/rootpy from upstream/master and did a git 
> diff and the output was the same as this commit... I'm confused. Is 
> there a simple place to change it back?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/BristolTopGroup/DailyPythonScripts/pull/318/files/7fd0f023e56f067c56c2f870132379dc3f3a1b65..d4d947958b557a9806e8a3e087984837432c8383#r60705456>
>